### PR TITLE
feat(); deployment.json; Add project tag to the deployment.json

### PR DESCRIPTION
**Pull request by steegecs**
**Context:**
@yulesa requested to have a project tag within the deployment.json. The project tag encompasses all related subgraphs under the same organization. For example, Aave V2, Aave V3, Aave RWA, Aave AMM, Aave ARC, and Aave Governance will receive the project tag of `aave`. Balancer V2 and Beethoven X will receive the project tag of `balancer`. Osmosis DEX and Osmosis chain will receive a tag of `osmosis.` This will be used to reference subgraphs by a project category in protocol metrics so that you can view all of the different facets of Aave, for example.

### CODE REVIEW

File: deployment/deployment.json [hunk 1]
```diff
@@ -1046,7 +1058,7 @@
   "belt-finance": {
     "schema": "yield-aggregator",
     "base": "belt-finance",
-    "protocol": "belt-finance",
+    "protocol": "belt",
```
Comment (dmelotik) [inline deployment/deployment.json:1061]: should say project, and bring back protocol

File: deployment/deployment.json [hunk 2]
```diff
@@ -1138,6 +1152,7 @@
     "schema": "lending",
     "base": "compound-forks",
     "protocol": "banker-joe",
+    "project": "banker-joe",
```
Comment (dmelotik) [inline deployment/deployment.json:1155]: trader-joe

File: deployment/deployment.json [hunk 3]
```diff
@@ -3090,6 +3138,7 @@
     "schema": "lending",
     "base": "maple-finance",
     "protocol": "maple-finance",
+    "project": "maple-finance",
```
Comment (dmelotik) [inline deployment/deployment.json:3141]: maple

File: deployment/deployment.json [hunk 4]
```diff
@@ -4065,6 +4130,7 @@
     "schema": "yield-aggregator",
     "base": "rari-vaults",
     "protocol": "rari-vaults",
+    "project": "rari",
```
Comment (dmelotik) [inline deployment/deployment.json:4133]: rari-capital

File: deployment/deployment.json [hunk 5]
```diff
@@ -4595,6 +4668,7 @@
     "schema": "dex-amm",
     "base": "uniswap-forks",
     "protocol": "trader-joe",
+    "project": "joe",
```
Comment (dmelotik) [inline deployment/deployment.json:4671]: trader-joe

File: deployment/deployment.json [hunk 6]
```diff
@@ -4901,6 +4981,7 @@
     "schema": "governance",
     "base": "maker-governance",
     "protocol": "maker-governance",
+    "project": "maker",
```
Comment (dmelotik) [inline deployment/deployment.json:4984]: makerdao

File: deployment/deployment.json [hunk 7]
```diff
@@ -7116,7 +7252,7 @@
   "orbit-bridge": {
     "schema": "bridge",
     "base": "orbit-bridge",
-    "protocol": "orbit-bridge",
+    "protocol": "orbit",
```
Comment (dmelotik) [inline deployment/deployment.json:7255]: keep protocol, add project

File: deployment/deployment.json (hunk 6 above)
Comment (steegecs) [reply deployment/deployment.json:4984]: agreed

File: deployment/deployment.json (hunk 7 above)
Comment (steegecs) [reply deployment/deployment.json:7255]: fixed

File: deployment/deployment.json (hunk 5 above)
Comment (steegecs) [reply deployment/deployment.json:4671]: Yup

**Review by dmelotik — APPROVED** [review]
lgtm!

**Pull request merged**